scripts/flash/fill-market-orders.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var async = require("async");
+var chalk = require("chalk");
+var fillOrder = require("../dp/lib/fill-order");
+
+function fillMarketOrderInternal(augur, marketIds, orderType, outcomeId, universe, auth, callback) {
+  augur.markets.getMarkets({ universe: universe, sortBy: "endDate", isSortDescending: true }, function (err, marketIds) {
+    if (!marketIds || marketIds.length === 0) {
+      console.log(chalk.red("No markets available"));
+      callback("No Markets");
+    }
+    augur.markets.getMarketsInfo({ marketIds: marketIds }, function (err, marketInfos) {
+      if (err) {
+        console.log(chalk.red("Error "), chalk.red(err));
+        return callback(err);
+      }
+      if (!marketInfos || !Array.isArray(marketInfos) || !marketInfos.length) {
+        return callback("No Market Info");
+      }
+      var market = marketInfos[0];
```

Comment:
Only does the first one? Not sure what the expected behavior for this script is

Reply:
Only takes in one market. No endpoint that takes a single marketId. See the fillMarketOrderIneternal call.

Reply:
OK, makes sense. So i think the issue is at the top of this function, the passed in market list isn't being used at all, and it just grabs all markets sorted by endDate.

Reply:
I'll clean up the marketId by removing the initial getMarkets(...) call, it's not needed, another copy paste typo 🤕